### PR TITLE
Update tribler_window.py call setWindowIcon

### LINKED ISSUE #5606: Running app has no icon
#### Please, fill all relevant items: ####

- [x] I have NOT read CONTRIBUTING.rst (no such file)
- [x] I have tried with the latest pre-release version and I still can reproduce the issue. (no prerelease)

##### Tribler version/branch+revision: #####
7.5.2

##### Operating system and version: #####
* Windows 10.2004
* Fedora Linux 32, KDE

##### Steps to reproduce the behavior: #####
Install Tribler and run it.

##### Expected behavior: #####
Icon in the top left corner of the title bar. Icon on the task manager.

##### Actual behavior: #####
No icon.

##### Relevant screenshots: #####
Screenshot of missing icons

### PR DESCRIPTION
**Pull request by mouse-the**
Call setWindowIcon() to get rid of the default icon...

Under debian/mate tribler has a default icon in the task panel, and it was bugging me. I have no experience on other platforms.

This is my first day on github, and I have not learned my way around tribler, yet. There is probably a better place to put this, but it works for me as is. It seems pretty safe. I stole the icon from the system tray code.

Fixes #5606

### CODE REVIEW

**Comment by qstokkink** [comment]
ok to test

**Comment by devos50** [comment]
Thanks! We should probably also test whether this will fix the Windows task manager icon, but I think it does. Thank you for your contribution!

Before we can merge this, can you 1) rebase your PR on the latest commit on the `devel` branch, and 2) fix the following pylint error?

```
141: C0303: (trailing-whitespace), : Trailing whitespace
```

Fixes https://github.com/Tribler/tribler/issues/5606

**Comment by mouse-the** [comment]
Hmmm, I seem to have accidentally merged devel into my branch while I was trying to figure out how to rebase on the website. Now I am not sure if I should still try to rebase.

**Comment by devos50** [comment]
Yeah I noticed. There now is a 'merge commit in your history. I recommend to do a rebase through the command-line interface with something like: `git rebase upstream/devel` (I added `upstream` as a separate git remote that points to https://github.com/tribler/tribler). Then I force push to my branch. For some practice, could you try again to rebase your PR, and squash all the commits to a single one?

Before we can merge this PR, I also have to check if this PR solves the task manager icon on Windows.

**Comment by mouse-the** [comment]
I think there is some chance I have it cleaned up now. I made it extra fun by starting out on the website and switching to the command line. Next time it should be easier, just follow https://tribler.readthedocs.io/en/devel/contributing.html

**Comment by devos50** [comment]
Confirmed to work on Windows and to fix #5606!

**Review by devos50 — APPROVED** [review]

**Pull request merged**
